### PR TITLE
Feature/lift

### PR DESCRIPTION
**Pull request by jacksondm33**

### CODE REVIEW

File: components/low/lift.py [hunk 1]
```diff
@@ -12,24 +12,23 @@ class Lift:
     lift_drive_right: ctre.WPI_VictorSPX
     lift_prox_front: wpilib.DigitalInput
     lift_prox_back: wpilib.DigitalInput
-    lift_limit_front_top: wpilib.DigitalInput
-    lift_limit_front_bottom: wpilib.DigitalInput
-    lift_limit_rear_top: wpilib.DigitalInput
-    lift_limit_rear_bottom: wpilib.DigitalInput
-    
+    lift_limit_front: wpilib.DigitalInput
+    lift_limit_back: wpilib.DigitalInput
```
Comment (CheezBallzPi) [inline components/low/lift.py]: There's a halleffect here too.

Comment (jacksondm33) [reply components/low/lift.py]: These two "limits" are just the two halleffects

Comment (CheezBallzPi) [reply components/low/lift.py]: I see.

**Comment by CheezBallzPi** [comment]
@jacksondm33 It's your branch, I'll let ya merge when you want.

**Pull request merged**